catroid/src/org/catrobat/catroid/content/Look.java

```diff
@@ -254,6 +255,10 @@ public LookData getLookData() {
 	public void setLookData(LookData lookData) {
 		this.lookData = lookData;
 		imageChanged = true;
+
+		if (this.equals(ProjectManager.getInstance().getCurrentProject().getSpriteList().get(0).look)) {
```

Comment:
you are right! I changed it. That way, the functionality is not susceptible to changes in the ``Project`` class. (Which are likely to happen when the Scenes feature has been merged)